HARK/ConsumptionSaving/ConsWealthPortfolioModel.py

```diff
@@ -559,10 +474,6 @@ def solve_one_period_WealthPortfolio(
     # PARAMETERS REQUIRED TO SIMULATE THE MODEL
     "AgentCount": 10000,  # Number of agents of this type
     "T_age": None,  # Age after which simulated agents are automatically killed
-    "aNrmInitMean": 0.0,  # Mean of log initial assets
-    "aNrmInitStd": 1.0,  # Standard deviation of log initial assets
-    "pLvlInitMean": 0.0,  # Mean of log initial permanent income
-    "pLvlInitStd": 0.0,  # Standard deviation of log initial permanent income
     "PermGroFacAgg": 1.0,  # Aggregate permanent income growth factor
```

Comment:
These simulation init parameters were removed. Are they now inherited from a parent class? If the parent defaults differ, this could cause unexpected behavior in simulations.